### ISSUE TITLE
muxer.finalize throws RangeError unless add chunk at least once

### ISSUE THREAD

**Issue by fifzteen** [Vanilagy/webm-muxer#53]
Hi. This library has helped me a lot. thanks 😄 
I found muxer.finalize() throws RangeError unless add videoChunk at least once.

sample code:

```js
import { Muxer, FileSystemWritableFileStreamTarget } from 'webm-muxer';

let fileHandle = await window.showSaveFilePicker({
    suggestedName: `video.webm`,
    types: [{
        description: 'Video File',
        accept: { 'video/webm': ['.webm'] }
    }],
});
let fileStream = await fileHandle.createWritable();
let muxer = new Muxer({
    target: new FileSystemWritableFileStreamTarget(fileStream),
    // ...
});
let videoEncoder = new VideoEncoder({
    output: (chunk, meta) => muxer.addVideoChunk(chunk, meta),
    error: e => console.error(e)
});
videoEncoder.configure(/* ... */);

let videoFrames = /* generator yields VideoFrame */
for await (const f of videoFrames) {
  /* cancelling encode => addChunk (e.g. cancelled by user) */
  // encode.encode(f)
}

muxer.finalize();
await fileStream.close();
```

get Error below.

```console
RangeError: Array buffer allocation failed
    at new ArrayBuffer (<anonymous>)
    at new Uint8Array (<anonymous>)
```

How should I handle this error? (can I ignore or skip finalize...?)

**Comment by Vanilagy** [comment]
Hey,

thanks for the report, I was able to reproduce it. This library is intended to support files with zero samples, so this is definitely something that I'll need to look into and fix. I'll investigate a fix once I have the time.

**Comment by Vanilagy** [comment]
Fixed in version 5.0.3.

**Comment by fifzteen** [comment]
5.0.3 works fine ! thank you for fixing !!!
